contributed_definitions/NXcoordinate_system.nxdl.xml

```diff
@@ -140,4 +140,11 @@ https://www.zenodo.org/record/3526738/files/lyso009a_0087.JF07T32V01_master.h5?d
             <dim index="1" value="3"/>
         </dimensions>
     </field>
+    <field name="depends_on" type="NX_CHAR"/>
```

Comment:
I tend to be against this design choice here to define an NXcoordinate_system again via NXtransformations as can be used to define a circularity.
An NXtransformation is essential a convenient way to describe the action of a tensor.
That tensor has to live in some reference frame ,depends_on could be used to make this reference frame explicit because NXcoordinate_system now allows to use something else than McStas.
Because of releasing this implicit old constraint of one has to use McStas the situation becomes more complicated. depends_on makes clear where the tensor is living in.

Therefore, I support that NXtransformation has a depends on
Therefore, I support that one chain (the action) of transformations with the chain demanding a clear definition of an end i.e. via the string "." AND the chain having an explicit mentioning in which reference frame you are working.
Allowing however to have again instances of NXtransformations building NXcoordinate_system is problematic because what happens if these NXtransformations (in the goodwill to define e.g. the three base vectors) again use depends_on that point somewhere else? As this could go wild I think we should not use it.

The discussion about passive and active transformations cannot be avoided when talking about coordinate systems and translations the discussion just shifts to us defining crystal-clear doc strings.